vg/vg.go

```diff
@@ -85,6 +88,49 @@ type CanvasSizer interface {
 	Size() (x, y Length)
 }
 
+// CanvasWriterTo wraps behavior required for writing
+// a Canvas representation to a concrete image.
+type CanvasWriterTo interface {
+	CanvasSizer
+	io.WriterTo
+}
+
+var (
+	formatLock sync.RWMutex
+	formats    = make(map[string]func(w, h Length) CanvasWriterTo)
```

Comment:
make it:
```go
func(w,h Length, title string) ...
```
instead? so eps, x11 and possibly others (opengl, html5/d3) can be "properly" called?